### PR TITLE
#165480893 Add credit account functionality

### PR DESCRIPTION
**Pull request by halimahO**
#### What does this PR do?
Implements credit account functionality

#### Description of Task to be completed?
Have the endpoint below working

POST api/v1/transactions/:accountnumber/credit

#### How should this be manually tested?
* clone the repo git clone https://github.com/halimahO/Banka.git

* on your terminal run cd Banka

* run npm install

* run npm run startDev

* make a request to POST localhost:3000/api/v1/transactions/:accountnumber/credit

#### What are the relevant pivotal tracker stories?
#165480893

### CODE REVIEW

File: server/controllers/accountController.js [hunk 1]
```diff
@@ -21,9 +19,9 @@ export default class AccountController {
     const { accountnumber } = req.params;
     const accountExists = await Account.getAccount(accountnumber);
     if (!accountExists) {
-      return res.status(404).json({
-        status: 404,
-        error: 'Account number not found.',
+      return res.status(400).json({
+        status: 400,
+        error: 'Account does not exist.',
```
Comment (kampkelly) [inline server/controllers/accountController.js]: This should be status 404. Why did you change it to 400?

File: server/controllers/accountController.js [hunk 2]
```diff
@@ -52,9 +50,9 @@ export default class AccountController {
 
     const accountExists = await Account.getAccount(accountnumber);
     if (!accountExists) {
-      return res.status(404).json({
-        status: 404,
-        error: 'Account number not found.',
+      return res.status(400).json({
+        status: 400,
+        error: 'Account does not exist.',
       });
```
Comment (kampkelly) [inline server/controllers/accountController.js]: This should be status 404. Why did you change it to 400?

File: server/controllers/transactionController.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default class TransactionController {
     }
 
     if (balance < transaction.amount) {
-      res.status(401).json({
+      return res.status(401).json({
         status: 401,
         error: 'Insufficient fund',
```
Comment (kampkelly) [inline server/controllers/transactionController.js]: This status code is incorrect.

File: server/controllers/accountController.js (hunk 2 above)
Comment (halimahO) [reply server/controllers/accountController.js]: Sorry, I changed it back to 404. I automatically changed the status code with the error message without thinking.

File: server/controllers/transactionController.js (hunk 3 above)
Comment (halimahO) [reply server/controllers/transactionController.js]: I changed it to 400

File: server/controllers/accountController.js (hunk 1 above)
Comment (halimahO) [reply server/controllers/accountController.js]: Sorry, I changed it back to 404.
